3rd-party/woocommerce-services.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class WC_Services_Installer {
+
+	public function init() {
+		$this->add_error_notice();
+		$this->try_install();
+	}
+
+	public function try_install() {
+		$action = filter_input( INPUT_GET, 'wc-services-action' );
```

Comment:
In this case, we only care about it if it's 'install', so it'd probably be simpler to combine the two lines into

```php
if ( isset( $_GET['wc-services-action'] ) && 'install' === $_GET['wc-services-action'] ) {
```

and would be more in keeping with the existing codebase for readability.